### PR TITLE
replace ie and eg with written out forms

### LINKED ISSUE #1974: Remove `e.g.` and `i.e` from the docs
- [x] We should replace `e.g.` with `for example` and `i.e.` with `meaning` or `that is` as everyone does not understand them
 - [ ] Create a Vale rule to throw an error when `e.g.` or `i.e.` is used

### PR DESCRIPTION
**Pull request by sixhobbits**
# Pull Request

## Related issue
* Fixes #1974 
* does not include vale rules, but replaces existing occurrences

## What does this PR do?
* replaces i.e. and e.g. with their written out forms as not everyone understands Latin abbreviations.

## PR checklist
Please check if your PR fulfills the following requirements:
- [ x] Does this PR fix an existing issue, or have you listed the changes applied in the PR description (and why they are needed)?
- [ x] Have you read the contributing guidelines?
- [ x] Have you made sure that the title is accurate and descriptive of the changes?

Thank you so much for contributing to Meilisearch!

### CODE REVIEW

File: learn/what_is_meilisearch/comparison_to_alternatives.md [hunk 1]
```diff
@@ -229,19 +229,19 @@ Due to its extensible plugin architecture and customizable features, Solr is a s
...
 
 ### Closed source
 
 #### Algolia
 
-Algolia is a company providing a search engine on a SaaS model. Its software is closed source. In its early stages, Algolia offered mobile search engines that could be embedded in apps, facing the challenge of implementing the search algorithms from scratch. From the very beginning, the decision was made to build a search engine directly dedicated to the end-users, i.e., implementing search within mobile apps or websites.
+Algolia is a company providing a search engine on a SaaS model. Its software is closed source. In its early stages, Algolia offered mobile search engines that could be embedded in apps, facing the challenge of implementing the search algorithms from scratch. From the very beginning, the decision was made to build a search engine directly dedicated to the end-users, that is, implementing search within mobile apps or websites.
```
Comment (maryamsulemani97) [inline learn/what_is_meilisearch/comparison_to_alternatives.md:244]:
```suggestion
Algolia is a company providing a search engine on a SaaS model. Its software is closed source. In its early stages, Algolia offered mobile search engines that could be embedded in apps, facing the challenge of implementing the search algorithms from scratch. From the very beginning, the decision was made to build a search engine directly dedicated to the end-users, specifically, implementing search within mobile apps or websites.
```

**Review by maryamsulemani97 — CHANGES_REQUESTED** [review]
Hi @sixhobbits 👋 
Thank you for your PR! I made some minor changes as `that is` and `for example` didn't sound right in some sentences. We should have clarified that in the [issue](https://github.com/meilisearch/documentation/issues/1974) 🙏 
Please let me know if you have any questions.

Thanks!

**Comment by sixhobbits** [comment]
thanks @maryamsulemani97 and @dichotommy - glad this one was useful. Requested changes have been committed. 

at Ritza.co we love helping companies extend and improve their docs, so just shout if we can ever be of more help!

**Review by maryamsulemani97 — APPROVED** [review]
Thanks again @sixhobbits!

bors merge

**Pull request merged**